vispy/visuals/image.py

```diff
@@ -400,8 +400,14 @@ def _build_texture(self):
             # deal with clim on CPU b/c of texture depth limits :(
             # can eventually do this by simulating 32-bit float... maybe
             clim = self._clim
+            # deal with masked_array
+            if np.ma.is_masked(data):
+                mask = data.mask
+                data = np.array(data)
```

Comment:
Does this make a copy of the array?

Reply:
Are you talking about the `np.ma.is_masked(data)`? If yes, no, it only return a bool indicating if the array has a `mask` attribute.

Reply:
No, I meant the `np.array` on a masked array.

Reply:
Yes, it returned a copy of the array but not `data.data`

Reply:
So it is a view of the array and does **not** copy the data to another memory location?

Reply:
Yes, it's a [view of the array](https://docs.scipy.org/doc/numpy-1.13.0/reference/maskedarray.generic.html#accessing-the-data)